---
content_type: page
description: This section contains descriptions of topics that were discussed during
  each class meeting.
draft: false
title: Lecture Summaries
uid: 6e143a9b-0729-4cfd-a211-6267174c0265
---
Each Lecture Summary is based on the corresponding {{% resource_link "1a51e15f-8d5c-4fe7-8614-7aa08e847ead" "Readings" %}} for that week.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
Week 
{{< thclose >}}{{< thopen >}}
Topics
{{< thclose >}}{{< thopen >}}
Lecture Summaries
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
1
{{< tdclose >}}{{< tdopen >}}
Introduction
{{< tdclose >}}{{< tdopen >}}
In this session we will introduce ourselves and go over the course syllabus. The goals, grading, and expectations will be covered. Students should be prepared to briefly describe their backgrounds, why they have chosen to take the course and what they expect to learn from it. An introduction to the basic principles of directed evolution will be discussed. Students will be introduced to the general format of the scientific literature and how to access the primary research literature using PubMed, SciFinder®, Google Scholar, and MIT library resources. We will briefly discuss the basics of directed evolution so that students have appropriate base knowledge to critically read assigned papers. To introduce the next week’s session, we will provide a brief overview of in vitro gene diversification techniques as a first step in directed evolution workflows.
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
2
{{< tdclose >}}{{< tdopen >}}
*In vitro* Gene Diversification.
{{< tdclose >}}{{< tdopen >}}

Genetic diversity fuels both natural and laboratory evolution. While the occurrence of mutations that introduce diversity is slow in nature, methods have been developed to rapidly introduce mutations in the laboratory. Targeted or focused mutagenesis is useful for biomolecules with appropriately characterized structures to guide mutations at specific locations. Random mutagenesis provides a broader outcome of diverse biomolecules. In this session we will discuss *in vitro*\-based mutagenesis methods to generate large gene libraries for directed evolution. 

In the first paper we will discuss, the Schwaneberg group develops the sequence saturation mutagenesis (SeSaM) technique, wherein DNA fragments of random lengths are generated, elongated with a universal non-natural base, further elongated to provide the full length gene, and finally amplified to replace the non-natural base. With this method, random mutations were observed in a manner that is complementary to error-prone PCR. 

The second paper we will discuss leverages a biological phenomenon of DNA repair called nonhomologous recombination, where broken DNA strands are joined together without the need for a template. The Liu group developed a nonhomologous random recombination (NRR) technique to diversify a gene using DNA strands with no homology or ordering requirements. This simple technique provides a greater level of diversity, enabling aptamer and protein evolution.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
3
{{< tdclose >}}{{< tdopen >}}
*In vivo* Gene Diversification.
{{< tdclose >}}{{< tdopen >}}

This session will focus on *in vivo* methods that introduce diversity to generate libraries suitable for directed evolution. One method leverages a component of the mammalian adaptive immune system, somatic hypermutation, where in the presence of an antigen, antibody (Ig) genes are mutated along specific regions. This principle was employed by Wang et al. to mutate nonantibody genes in mammalian cells, generating a library of fluorescent proteins by the addition of doxycycline antigen. 

Another paper we will discuss uses yeast for the *de novo* assembly of gene libraries. *Saccharomyces cerevisiae* can take up small, single strands of DNA and piece them into larger genomic assemblies using homologous recombination. Delivering predefined segments of DNA to yeast generates gene diversity through various instances of homologous recombination.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
4
{{< tdclose >}}{{< tdopen >}}
Identifying Desired Phenotypes -- Selection Methods
{{< tdclose >}}{{< tdopen >}}

The other core principle of directed evolution is the enrichment of desired traits. This is achieved in the laboratory through judicious choice of screening methods or tightly controlled selection parameters. Ideal clones, or proteins that exhibit a desired phenotype, must be isolated from the remaining population so that they can be enriched and continue the evolutionary process. 

In session #4 we will discuss papers that have developed selection methods for enrichment during directed evolution procedures. In one paper, Kaltenbach et al. utilize a dendritic DNA scaffold to display multiple copies of a protein. This system uses droplets to compartmentalize single encoding DNA molecules, and subsequently express multiple protein copies appended to the encoding DNA. Selection is based on stronger, multivalent interactions during binding events. 

In another method, Lin et al. develop a system in yeast where the activity of an enzyme of interest is made essential for cell growth. When the substrate is added, enzyme activity reconstitutes a transcriptional pathway needed for cell survival. A simple growth/death selection is employed to evolve a key enzyme in carbohydrate synthesis. 

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
5
{{< tdclose >}}{{< tdopen >}}
Identifying Desired Phenotypes -- Screening Methods
{{< tdclose >}}{{< tdopen >}}

In the second installment of Identifying Desired Phenotypes, we will discuss two papers that focus on the development of screening methods during directed evolution. Screening involves the identification and isolation of a desired clone. While the screening landscape is vast, Session #5 will focus on the use of flow cytometry and mass spectrometry methods. 

The Hollfelder group combined two techniques, droplet microfluidics and fluorescence activated cell sorting (FACS), to develop a high-throughput screening assay for enzyme evolution. *E. coli* cells expressing an arylsulfatase enzyme are individually compartmentalized into water-in-oil microdroplets, then lysed for analysis of activity by FACS. The compartmentalization into droplets for FACS offers many benefits in the directed evolution of proteins, primarily providing a genotype-phenotype linkage for proteins that cannot be screened by other means. 

In the second paper we will discuss, the Sweedler group screens using mass spectroscopy, a powerful tool for the characterization of various molecules based on mass-to-charge ratio. They developed a mass spectroscopy-based screening method to directly analyze microbial colonies for multistep pathway variants.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
6
{{< tdclose >}}{{< tdopen >}}
Continuous Evolution
{{< tdclose >}}{{< tdopen >}}

Continuous evolution (CE) aims to enable gene diversification, expression, and screening or selection to occur iteratively without human intervention. It can shorten the experimental time and increase the total number of rounds of evolution, thus largely enhancing evolution effectiveness in an evolutionary search.

This class session focuses on the foundational paper that describes the M13 bacteriophage-assisted continuous evolution (PACE) technique, as well as a novel technique that utilizes engineered adenovirus as a vector for continuous mutagenesis and selection in human cells.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
7
{{< tdclose >}}{{< tdopen >}}
Addressing Climate Change
{{< tdclose >}}{{< tdopen >}}
Climate change is driven by increasing atmospheric CO2 concentrations, and plants and algae are limited in their natural ability to absorb CO2 through photosynthesis. Addressing the fundamental limitations of CO2 capture is complex, and directed evolution strategies can be applied in a variety of ways. Nature’s central photosynthesis catalyst Rubisco converts atmospheric CO2 into sugar. New scaffolds for Rubisco can be generated using ancestral protein reconstruction to then be used as starting points for evolution. Other groups are seeking to reinvent the Calvin Cycle of photosynthesis by creating and evolving entirely new CO2 fixation catalysts.
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
8
{{< tdclose >}}{{< tdopen >}}
Engineering Waste Solutions
{{< tdclose >}}{{< tdopen >}}

(Please note: the field trip applies only to the on-campus course taught at MIT.) This class session will be a virtual field trip to a local lab (Zymergen), hosted by Dr. Caitlin Allen. This will be a ~45 min presentation titled, "The Next Frontier of the Bioeconomy and Evolution in Chemical Engineering", followed by a virtual lab tour showing some instruments and brief procedures they use. 

Zymergen is a biotechnology company that utilizes organisms to develop solutions in materials, electronics, agriculture, and waste. Initially the Boston location was born as enEvolv, a startup founded by George Church, Farren Isaacs, and Jay Konieczka, that recently merged with parent company Zymergen. Caitlin has been with the Boston location through the many phases of inception and growth, and can provide realistic insight into many aspects of this process. This is a good opportunity to inquire not only about directed evolution applications in industry, but also about obtaining a job and the company life cycle in this area of industry.

If there is time after the virtual field trip we may quickly discuss the assigned article, the 2021 ChemRxiv paper on PETase. The products of human activities are often environmental pollutants (plastics, industrial chemicals, oil spills, heavy metals). These chemicals accumulate in the environment often because there are few biological processes that can efficiently break them down. Scientists strive to apply directed evolution techniques to develop new enzyme functionalities that can support this role and permit living organisms to utilize these chemicals or break them down into non-toxic products. For unnatural waste, such as plastics, new enzymes must be developed. In a preprint, Bell et al. 2021 evolve a thermostable polyethylene terephthalate (PET) depolymerase for the industrial breakdown of plastics. 

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
9
{{< tdclose >}}{{< tdopen >}}
Developing Therapeutics
{{< tdclose >}}{{< tdopen >}}

Progress in human health is one of the core missions of scientific research. Over many years this has resulted in successful therapies utilizing small molecules, proteins, genes, and cells. With initial successes, researchers have since incorporated directed evolution techniques to push these therapies even further. Binding proteins, such as antibodies, antibody fragments, transcription factors and more, have been evolved to exhibit greater stability and activity in their therapeutic purposes. Viral vectors and nonviral protein cages are evolved for greater gene therapy delivery systems. Meanwhile whole cell evolution for cell-based therapeutics has been comparatively lacking. 

In this session we will discuss two papers that highlight recent developments in therapeutically-focused directed evolution. The Wittrup lab leveraged temperature-selective screening methods to develop a highly stable, single chain protein from the surface of immune cells (T-cells) that exhibits key interactions with another adaptive immune protein. The evolved proteins have diverse potential therapeutic applications such as virus inhibition and cancer cell elimination. 

Gray et al 2021 utilize scanning unnatural protease resistant (SUPR) mRNA display to generate cyclic peptides that inhibit upregulated cell degradation in treatment resistant cancers.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
10
{{< tdclose >}}{{< tdopen >}}
Understanding Evolutionary Processes
{{< tdclose >}}{{< tdopen >}}

“Evolution is cleverer than you are” - Leslie Orgel. Evolution finds a way - but it’s usually a way we do not expect or, as scientists, are able to adequately anticipate during our experimental design. Understanding how evolutionary processes work is critical to designing robust directed evolution campaigns and is advantageous for predicting the evolution of pathogens in response to therapeutics. 

In Nyerges et al., 2018, the directed evolution with random genomic mutations (DIvERGE) technique is used to develop clinically significant resistance against the antibiotics trimethoprim and ciprofloxacin in multiple bacterial species and generates a high level of resistance against gepotidacin, a novel antibiotic currently in clinical trials. 

Morganthaler et al. 2019 demonstrate how evolution will often find compensatory routes to improve host fitness rather than directly improve an inefficient target or enzymatic step within a metabolic process. These off-target effects are difficult to predict and can derail attempts to enhance a specific biomolecule of interest.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
11
{{< tdclose >}}{{< tdopen >}}
Next Generation Techniques in Directed Evolution: Evolution Beyond the Genetic Code and Natural Starting Points
{{< tdclose >}}{{< tdopen >}}
As you have learned in this course, it is routine for scientists to seek improved or altered functions from existing enzymes in Nature using a wide variety of tools. How can we push these boundaries further? Researchers have developed, and are improving, methods for life to incorporate totally new amino acids and chemistries into enzymes as they evolve, opening the potential for functions that otherwise could never exist naturally. Natural enzymes are the product of their entire evolutionary lineage and limit what new functions can be built upon these scaffolds. Entirely new (*de novo*) production of enzymes can potentially break this constraint and enable the evolution of familiar and alien chemistries.
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
12
{{< tdclose >}}{{< tdopen >}}
Next Generation Techniques in Directed Evolution: Evolving Biosynthetic Pathways and Leveraging Machine Learning in Directed Evolution
{{< tdclose >}}{{< tdopen >}}

Pushing the boundaries differently than discussed in Session #11, researchers are tackling more complex systems and leveraging predictive power to aid in the directed evolution process. In this session we will focus on a paper that improves the multi-enzyme biosynthetic pathway towards an important chemical building block, and the development of a machine learning method that predicts high-fitness variants to guide evolution in the laboratory.

Following this, we will be having a discussion on directed evolution with Prof. Ahmed Badran from Scripps Research (San Diego), via Zoom. Ahmed was a graduate student in the Liu lab making the assigned paper even more topical. Ahmed will give a presentation on his career path in academia and where the field of directed evolution is headed. 

You can out more about his research at the lab website: {{% resource_link "9da13648-0f7e-4f96-856c-7a7858e7a239" "https://badranlab.com/" %}}

(Please note: the guest discussion with Prof. Badran applies only to the on-campus course taught at MIT.)

We will not be discussing the other 'pathway evolution' paper.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
13
{{< tdclose >}}{{< tdopen >}}
Final Presentations
{{< tdclose >}}{{< tdopen >}}
Students will give oral presentations (~15 min talk + 10 min discussion) about their chosen research papers and lead critical discussions about the papers’ findings. We will have a final discussion about what we have learned about directed evolution, and the resulting technologies that have arisen from this technique. Students will complete a course evaluation and provide feedback to the instructors about the course.
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}