---
content_type: page
description: 'This section contains a list of journal articles that the students were
  expected to read before each class '
draft: false
title: Readings
uid: 1a51e15f-8d5c-4fe7-8614-7aa08e847ead
---
See the {{% resource_link "6e143a9b-0729-4cfd-a211-6267174c0265" "Lecture Summaries" %}} to read about the discussions that took place around each week's readings.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
Week 
{{< thclose >}}{{< thopen >}}
Topics
{{< thclose >}}{{< thopen >}}
Readings
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
1
{{< tdclose >}}{{< tdopen >}}
Introduction
{{< tdclose >}}{{< tdopen >}}

There are **no required readings for the first session**, but below are some links to short introductory videos for those interested. And (still optional) a technical review paper for a more in-depth introduction. We recommend at least looking at the figures, as the full text may take a lot of time.

Chemical & Engineering News. "[The 2018 Nobel Prize in Chemistry: Directed evolution & phage display--Speaking of Chemistry](https://www.youtube.com/watch?v=ZKfRWqGtBDY&t=10s&ab_channel=Chemical%26EngineeringNews)." October 4, 2018. YouTube.  

iGEM at Stony Brook. "[Directed Evolution: Queens iGEM & Stony Brook iGEM Collaboration](https://www.youtube.com/watch?v=tSN4QHwr8G8&ab_channel=iGEMatStonyBrook)." Sept. 30, 2018. YouTube. 

The Nobel Prize. "[Frances H. Arnold: Nobel Lecture in Chemistry 2018](https://www.youtube.com/watch?v=6hOZ5e0g9Uo&ab_channel=NobelPrize)." Dec. 8, 2018. YouTube. 

Packer, M.S., and Liu, D.R. [Methods for the directed evolution of proteins](https://www.nature.com/articles/nrg3927). *Nature Reviews Genetics.* **2015**. 16, 379–394.   

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
2
{{< tdclose >}}{{< tdopen >}}
*In vitro* Gene Diversification.
{{< tdclose >}}{{< tdopen >}}

Wong, T.S., Tee, K.L., Hauer, B., and Schwaneberg, U. [Sequence saturation mutagenesis (SeSaM): a novel method for directed evolution](https://academic.oup.com/nar/article/32/3/e26/2904611). *Nucleic Acids Res*. **2004**. 32 (2): e26.

Bittker, J.A., Le, B.V., Liu, J.M., and Liu, D.R. [Directed evolution of protein enzymes using nonhomologous random recombination](https://www.pnas.org/doi/full/10.1073/pnas.0402202101). *PNAS.* **2004**. 101 (18), 7011–7016.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
3
{{< tdclose >}}{{< tdopen >}}
*In vivo* Gene Diversification.
{{< tdclose >}}{{< tdopen >}}

Wang, L., Coyt Jackson, W., Steinbach, P.A., and Tsien, R.Y. [Evolution of new nonantibody proteins via iterative somatic hypermutation](https://www.pnas.org/doi/full/10.1073/pnas.0407752101). *PNAS*. **2004**. 101 (48), 16745-16749.

DiCarlo, J.E., Conley, A.J., Penttila, M., Jantti, J., Wang, H.H., Church, G.M. [Yeast Oligo-Mediated Genome Engineering (YOGE)](https://pubs.acs.org/doi/10.1021/sb400117c). *ACS Synthetic Biology*. **2013.** 2(12), 741-749.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
4
{{< tdclose >}}{{< tdopen >}}
Identifying Desired Phenotypes -- Selection Methods
{{< tdclose >}}{{< tdopen >}}

Kaltenbach, M., Stein, V., and Hollfelder, F. [SNAP Dendrimers: Multivalent Protein Display on Dendrimer-Like DNA for Directed Evolution](https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.201100240). *ChemBioChem.* **2011**. 12, 2208-2216.

Lin, H., Tao, H., and Cornish, V. W. [Directed Evolution of a Glycosynthase via Chemical Complementation](https://pubs.acs.org/doi/10.1021/ja046238v). *J. Am. Chem. Soc.* **2004**. 126, 46, 15051-15059.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
5
{{< tdclose >}}{{< tdopen >}}
Identifying Desired Phenotypes -- Screening Methods
{{< tdclose >}}{{< tdopen >}}

Kintses, B., Hein, C., Mohamed, M.F., Fischlechner, M., Courtois, F., Laine, C., and Hollfelder, F. [Picoliter Cell Lysate Assays in Microfluidic Droplet Compartments for Directed Enzyme Evolution](https://www.sciencedirect.com/science/article/pii/S1074552112002165?via%3Dihub). *Chemistry & Biology.* **2012**. 19(8), 1001-1009.

Si, T., Li, B., Comi, T.J., Wu, Y., Hu, P., Wu, Y., Min, Y., Mitchell, D.A., Zhao, H., and Sweedler, J.V. [Profiling of Microbial Colonies for High-Throughput Engineering of Multistep Enzymatic Reactions via Optically Guided Matrix-Assisted Laser Desorption/Ionization Mass Spectrometry](https://pubs.acs.org/doi/10.1021/jacs.7b04641). *J. Am. Chem. Soc.* **2017**. 139(36), 12466-12473.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
6
{{< tdclose >}}{{< tdopen >}}
Continuous Evolution
{{< tdclose >}}{{< tdopen >}}

Esvelt, K.M., Carlson, J.C., and Liu, D.R. [A System For the continuous directed evolution of biomolecules](https://pubmed.ncbi.nlm.nih.gov/21478873/). *Nature.* **2011**. 472, 499-503.

Berman, C.M., Papa, L.J., Hendel, S. J., Moore, C.L., Suen, P.H., Weickhardt, A.F., Doan, N., Kumar, C.M., Uil, T.G., Butty, V.L., Hoeben, R.C., and Shoulders, M.D. [An Adaptable Platform for Directed Evolution in Human Cells](https://pubs.acs.org/doi/abs/10.1021/jacs.8b10937). *J. Am. Chem. Soc.* **2018**. 140(51), 18093-18103.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
7
{{< tdclose >}}{{< tdopen >}}
Addressing Climate Change
{{< tdclose >}}{{< tdopen >}}

Gomez-Fernandez, B. J., Garcia-Ruiz, E., Martin-Diaz, J., Gomez de Santos, P., Santos-Moriano, P., Plou, F. J., Ballesteros, A., Garcia, M., Rodriguez, M., Risso, V. A., Sanchez-Ruiz, J. M., Whitney, S. M., and Alcalde, M. [Directed -*in vitro*\- evolution of Precambrian and extant Rubiscos](https://www.nature.com/articles/s41598-018-23869-3). *Scientific Reports* **2018**. 8, 5532.

Scheffen, M., Marchal, D. G., Beneyton, T., Schuller, S.K., Klose, M., Diehl, C., Lehmann, J., Pfister, P., Carrillo, M., He, H., Aslan, S., Cortina, N. S., Claus, P., Bollschweiler, D., Baret, J.-C., Schuller, J. M., Zarzycki, J., Bar-Even, A., and Erb, T.J. [A new-to-nature carboxylation module to improve natural and synthetic CO2 fixation](https://www.nature.com/articles/s41929-020-00557-y). *Nature Catalysis* **2021**. 4, 105–115.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
8
{{< tdclose >}}{{< tdopen >}}
Engineering Waste Solutions
{{< tdclose >}}{{< tdopen >}}
Bell, E., Smithson, R., Kilbride, S., Foster, J., Hardy, F., Ramachandran, S., Tedstone, A., Haigh, S., Garforth, A., Day, P., Levy, C., Shaver, M., and Green, A. [Directed Evolution of an Efficient and Thermostable PET Depolymerase.](https://chemrxiv.org/engage/chemrxiv/article-details/61c9db381e13eb8ccc076210) *ChemRxiv*. **2021**.
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
9
{{< tdclose >}}{{< tdopen >}}
Developing Therapeutics
{{< tdclose >}}{{< tdopen >}}

Shusta, E.V., Holler, P.D., Kieke, M.C., Kranz, D.M., and Wittrup, D.K. [Directed evolution of a stable scaffold for T-cell receptor engineering](https://www.nature.com/articles/nbt0700_754). *Nature Biotechnology*. **2000**. 18, 754-759.

Gray, J.P., Uddin, M.N., Chaudhari, R., Sutton, M.N., Yang, H., Rask, P., Locke, H., Engel, B.J., Batistatou, N., Wang, J., Grindel, B.J., Bhattacharya, P., Gammon, S.T., Zhang, S., Piwnica-Worms, D., Kritzer, J.A., Lu, Z., Bast, R.C., and Milward, S.W. [Directed evolution of cyclic peptides for inhibition of autophagy](https://pubs.rsc.org/en/content/articlepdf/2021/sc/d0sc03603j). *Chem. Sci.* **2021**. 12, 3526-3543.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
10
{{< tdclose >}}{{< tdopen >}}
Understanding Evolutionary Processes
{{< tdclose >}}{{< tdopen >}}

Nyerges, Á., Csörgő, B., Draskovits, G., Kintses, B., Szili, P., Ferenc, G., Révész, T., Ari, E., Nagy, I., Bálint, B. and Vásárhelyi, B.M. [Directed evolution of multiple genomic loci allows the prediction of antibiotic resistance.](https://www.pnas.org/doi/full/10.1073/pnas.1801646115) *Proceedings of the National Academy of Sciences*, **2018**. 115(25), E5726-E5735.

Morgenthaler, A.B., Kinney, W.R., Ebmeier, C.C., Walsh, C.M., Snyder, D.J., Cooper, V.S., Old, W.M. and Copley, S.D. [Mutations that improve efficiency of a weak-link enzyme are rare compared to adaptive mutations elsewhere in the genome](https://elifesciences.org/articles/53535). *Elife*, **2019**. 8, e53535.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
11
{{< tdclose >}}{{< tdopen >}}
Next Generation Techniques in Directed Evolution: Evolution Beyond the Genetic Code and Natural Starting Points
{{< tdclose >}}{{< tdopen >}}

Reddington, S. C., Baldwin, A. J., Thompson, R., Brancale, A., Tippmann, E. M., and Jones, D. D. [Directed evolution of GFP with non-natural amino acids identifies residues for augmenting and photoswitching fluorescence](https://pubs.rsc.org/en/content/articlelanding/2015/sc/c4sc02827a). *Chem. Sci.* **2015**. 6, 1159–1166.

Wang, M. S., and Hecht, M. H. [A Completely De Novo ATPase from Combinatorial Protein Design](https://pubs.acs.org/doi/pdf/10.1021/jacs.0c02954). *JACS.* **2020**. 142, 15230–15234.

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
12
{{< tdclose >}}{{< tdopen >}}
Next Generation Techniques in Directed Evolution: Evolving Biosynthetic Pathways and Leveraging Machine Learning in Directed Evolution
{{< tdclose >}}{{< tdopen >}}

Shen, Y-P., Fong, L.S., Yan, Z-B., and Liu, J-Z. [Combining directed evolution of pathway enzymes and dynamic pathway regulation using a quorum-sensing circuit to improve the production of 4-hydroxyphenylacetic acid in *Escherichia coli.*](https://biotechnologyforbiofuels.biomedcentral.com/articles/10.1186/s13068-019-1438-3) *Biotechnology for Biofuels.* **2019**. 12, 94.

Shen, M.W., Zhao, K.T., and Liu, D. R. [Reconstruction of evolving gene variants and fitness from short sequencing reads](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8551035/). *Nat. Chem. Biol.* **2021**. 17, 1188-1198.

Optional Video: [Machine Learning Assisted Dir. Evo. of Viral Vectors and Microbial Opsins for Minimally Invasive Neuroscience](https://www.youtube.com/watch?v=7Wnz8UfzCU0) (30 min talk from Prof. Viviana Gradinaru at Caltech)

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
13
{{< tdclose >}}{{< tdopen >}}
Final Presentations
{{< tdclose >}}{{< tdopen >}}
No Readings
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}